src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.Helpers.cs

```diff
@@ -23,5 +23,17 @@ public static partial class JsonSerializer
 
             return state.Current.ReturnValue;
         }
+
+        private static void CheckSupportedOptions(
+            JsonSerializerOptions options,
+            string paramName = null)
```

Comment:
are you meaning to pass in "options" in the callers? Otherwise you can remove this parameter